### PR TITLE
Add additional routing daemons documentation 

### PR DESCRIPTION
**Pull request by rubensfig**

### CODE REVIEW

File: network_configuration/eigrp.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Enhanced Interior Gateway Routing Protocol (EIGRP)
+parent: Network Configuration
+---
+
+# Enhanced Interior Gateway Routing Protocol (EIGRP)
+
+## Introduction
+
+The Enhanced Interior Gateway Routing Protocol is a distance-vector routing protocol that is defined in [RFC 7868](https://tools.ietf.org/html/rfc7868). The protocol shares routes with other routers in the same Autonomous System. In contrary to other distance-vector protocols updates to the routing table is shared over differential updates, instead of transmitting the entire routing table. This reduces the control traffic on the network and provides faster convergence types for changes in network topology.
```
Comment (rubensfig) [inline network_configuration/eigrp.md:10]:
```suggestion
The Enhanced Interior Gateway Routing Protocol is a distance-vector routing protocol that is defined in [RFC 7868](https://tools.ietf.org/html/rfc7868). The protocol shares routes with other routers in the same Autonomous System. In contrary to other distance-vector protocols updates to the routing table is shared over differential updates, instead of transmitting the entire routing table. This reduces the control traffic on the network and provides faster convergence for changes in network topology.
```

File: network_configuration/eigrp.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Introduction
+
+The Enhanced Interior Gateway Routing Protocol is a distance-vector routing protocol that is defined in [RFC 7868](https://tools.ietf.org/html/rfc7868). The protocol shares routes with other routers in the same Autonomous System. In contrary to other distance-vector protocols updates to the routing table is shared over differential updates, instead of transmitting the entire routing table. This reduces the control traffic on the network and provides faster convergence for changes in network topology.
+
+**WARNING**: EIGRP is currently only tested for IPv4.
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/zebra.conf`.
```
Comment (jklare) [inline network_configuration/eigrp.md:16]: enabling the daemon is done in /etc/frr/daemons and zebra is always active (there is no zebra=yes option)

File: network_configuration/rip.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Introduction
+
+Routing Information Protocol (RIP) is one the oldest routing protocols available to use. While other routing protocols like IS-IS and OSPF are used in more modern applications due to reduced convergence time and improved scalability, RIP is one of the easiest protocols to configure, and small networks can take advantage of the small bandwidth usage and configuration/management simplicity.
+
+**WARNING**: EIGRP is currently only tested for IPv4.
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/zebra.conf`.
```
Comment (jklare) [inline network_configuration/rip.md:16]: enabling the daemon is done in /etc/frr/daemons and zebra is always active (there is no zebra=yes option)

File: network_configuration/rip.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+zebra=yes
+ripd=yes
+...
+vtysh_enable=yes
```
Comment (jklare) [inline network_configuration/rip.md:22]: these options are unrelated and there is no need to adapt them

File: network_configuration/eigrp.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+```
+zebra=yes
+eigrpd=yes
+...
+vtysh_enable=yes
```
Comment (jklare) [inline network_configuration/eigrp.md:22]: these options are unrelated and there is no need to adapt them

File: network_configuration/eigrp.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/daemons`.
+
+```
+eigrpd=yes
+...
+eigrpd_options=" --daemon -A 127.0.0.1"
+...
```
Comment (jklare) [inline network_configuration/eigrp.md:22]:
```suggestion
```

File: network_configuration/rip.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/daemons`.
+
+```
+ripd=yes
+...
+ripd_options="   --daemon -A 127.0.0.1"
+...
```
Comment (jklare) [inline network_configuration/rip.md:22]:
```suggestion
```

File: network_configuration/eigrp.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/daemons`.
+
+```
+eigrpd=yes
+```
+
+Then `/etc/frr/eigrpd.conf` can be configured as follows.
```
Comment (jklare) [inline network_configuration/eigrp.md:22]: i think it might be worth adding the full topology and the zebra.conf + eigrpd.conf from our tests here instead of this very compact config

File: network_configuration/rip.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+## FRR configuration
+
+Configuring EIGRP in FRR starts by activating the daemon in `/etc/frr/daemons`.
+
+```
+ripd=yes
+```
+
+Configuring the daemon is done via the `/etc/frr/ripd.conf`.
```
Comment (jklare) [inline network_configuration/rip.md:22]: i think it might be worth adding the full topology and the zebra.conf + ripd.conf from our tests here instead of this very compact config

**Comment by jklare** [comment]
lets merge this as a starting point and revise all routing docs in the next step
